DESCRIPTION

```diff
@@ -89,7 +90,9 @@ Remotes:
     MoTrPAC/MotrpacRatTraining6moData,
     MoTrPAC/MotrpacBicQC,
     YuLab-SMU/ggtree,
-    nicolerg/ssGSEA2
+    nicolerg/ssGSEA2,
+    cytoscape/RCy3,
```

Comment:
Download development version of `RCy3` to avoid `R-CMD check` error:
```
  * deps::.: Can't install dependency RCy3 (>= 2.23.2)
  * RCy3: Can't install dependency uchardet
  * uchardet: Can't find package called uchardet.
```
Oddly, the newest version of `RCy3` has already removed `uchardet` as a dependency, and I don't get an error installing `RCy3` locally, so I'm not sure why specifying the newest version isn't working for the GitHub Action. This is a temporary fix that should be reverted once the GitHub Action can pass.